sdk/core/core-lro/package.json

```diff
@@ -96,7 +96,7 @@
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-http": "^1.0.0",
-    "@opentelemetry/types": "^0.2.0",
```

Comment:
Why is opentelemetry even a dependency in core-lro? It doesnt seem to be in use in any of the source files

Reply:
I might have added it by copying from another place!

Reply:
#8387